Add note to adv network configuration docs w.r.t endpoint overlaps (#1147)
hazelcast/hz-docs: docs/modules/clusters/pages/network-configuration.adoc

WAN cluster members) or to establish server sockets on which a target WAN member can
listen for the incoming connections from the source cluster.

NOTE: When using the advanced network configuration and defining separate endpoints for

> `WAN` and `MEMBER` protocols, it is important to **ensure that endpoint ranges do not overlap**.

Reviewer: ```suggestion
`WAN` and `MEMBER` protocols, you must ensure that endpoint ranges do not overlap.
```

> Each protocol should be configured with unique endpoints that are not used in other protocols,

Reviewer: ```suggestion
Each protocol must be configured with unique endpoints that are not used in other protocols,
```

> otherwise WAN replication in environments where IPs are reused may cause the cluster to

Reviewer: ```suggestion
otherwise WAN replication can cause the cluster to
```

> become unstable.

Reviewer: ```suggestion
become unstable if IPs are reused.
```
